pincer/objects/message/user_message.py

```diff
@@ -539,3 +540,25 @@ async def delete(self):
         await self._http.delete(
             f"/channels/{self.channel_id}/messages/{self.id}"
         )
+
+    @property
+    def clean_content(self):
+        """
+        The message content with any special characters removed.
```

Comment:
``"""single line docstring"""`` its in the python docstring pep, also wheres the return typehint and docstrings.
docstrings for properties in numpy are:
```py
"""The[Type]: description"""
```